src/Algebra/Module/Bundles.agda

```diff
@@ -80,8 +77,17 @@ record LeftSemimodule (semiring : Semiring r ℓr) m ℓm
     ; magma     to +ᴹ-magma
     ; rawMagma  to +ᴹ-rawMagma
     ; rawMonoid to +ᴹ-rawMonoid
+    ; _≉_ to _≉ᴹ_
     )
 
+  rawLeftSemimodule : RawLeftSemimodule Carrier m ℓm
+  rawLeftSemimodule = record
+    { _≈ᴹ_ = _≈ᴹ_
+    ; _+ᴹ_ = _+ᴹ_
+    ; _*ₗ_ = _*ₗ_
+    ; 0ᴹ = 0ᴹ
+    }
```

Comment:
I would love it if such 'substructure' definitions were available without having to be written out in full, cf. #2255 ... eg here, the `+ᴹ-rawMonoid` structure is already available, only the additional `_*ₗ_` field needs defining, but Agda makes such `record` re-combination a bit crunchier than I would like:
```agda
rawLeftSemimodule = record { M ; _*ₗ_ = _*ₗ_ } where module M = RawMonoid +ᴹ-rawMonoid
```
is a bit of a mouthful, but still more 'informative' than the 'flattened' version?

Reply:
Would that work in this case? `RawMonoid` has different field names to `RawLeftSemimodule`...

Reply:
Ach! I think you're right. Maddening! 
Having to add the `renaming` directives would vitiate the whole suggestion. Grrr.